_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -478,19 +478,31 @@ def post(self, url: str, data: dict = {}):
 class FrontstageTasks(TaskSet, Mixins):
 
     def on_start(self):
-        self.sign_in()
+        requests_filepath = "requests.json"
 
-    def sign_in(self):
-        response = self.get(url="/sign-in", expected_response_text="Sign in")
-        self.csrf_token = _capture_csrf_token(response.content.decode('utf8'))
-
-        response = self.post("/sign-in", data=_generate_random_respondent())
-        self.auth_cookie = response.cookies['authorization']
+        with open(requests_filepath, encoding='utf-8') as requests_file:
+            requests_json = json.load(requests_file)
+            self.requests = requests_json["requests"]
 
     @task
-    def todo(self):
-        self.get("/surveys/todo", expected_response_text="Click on the survey name to complete your questionnaire")
+    def perform_requests(self):
+        for request in self.requests:
+            request_url = request['url']
+
+            if request["method"] == "GET":
+                request_response = request['expected_response_text']
```

Comment:
I personally would put it all on one line, if you want to leave it separate the name should reflect what it is, i.e the expected_response_text

Reply:
I changed the variable name to `expected_response_text`, but kept the variable assignment on its own line because I personally think it looks better that way.

Reply:
Done